telegram/ext/jobqueue.py

```diff
@@ -21,10 +21,22 @@
 import logging
 import time
 import warnings
+import datetime
+from enum import Enum
 from threading import Thread, Lock, Event
 from queue import PriorityQueue, Empty
 
 
+class Days(Enum):
```

Comment:
@bomjacob I've already done my research on this, [Python2](https://docs.python.org/2/library/datetime.html ) and [Python 3](https://docs.python.org/3/library/datetime.html) both return the .weekday() from 0-6 where monday is 0 and sunday is 6.